### PR TITLE
doc/website/blog: vis with atlas

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+authorTwitter: _rtam
+image: "https://atlasgo.io/uploads/ent/inspect/entviz.png"
+---
+
+### TL;DR
+
+Create a visualization of your Ent schema with one command:
+
+```
+atlas schema inspect --dev-url 'sqlite://demo?mode=memory&_fk=1' -u ent://ent/schema --visualize
```
Comment (a8m) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:15]:
```suggestion
atlas schema inspect \
  -u ent://ent/schema \
  --dev-url "sqlite://demo?mode=memory" \
  --visualize
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+```
+curl -sSfL https://atlasgo.io/install.sh | sh
+```
+For other installation options, see the [Atlas installation docs](https://atlasgo.io/getting-started#installation).
+
+Next, run the following command to generate a visualization of your Ent schema:
+
+```
+atlas schema inspect -u ent://ent/schema --dev-url 'sqlite://demo?mode=memory&_fk=1'  --visualize
```
Comment (a8m) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:55]:
```suggestion
atlas schema inspect \
  -u ent://ent/schema \
  --dev-url "sqlite://demo?mode=memory" \
  --visualize
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+Let's break this command down:
+* `atlas schema inspect` - this is command can be used to inspect schemas from a variety of sources and output
+  them in various formats. In this case, we are using it to inspect an Ent schema.
+* `-u ent://ent/schema` - this is the URL to the Ent schema we want to inspect. In this case, we are using the
+  `ent://` schema loader to point to a local Ent schema in the `./ent/schema` directory.
+* `--dev-url 'sqlite://demo?mode=memory&_fk=1'` - Atlas relies on having an empty database called the
```
Comment (a8m) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:63]:
```suggestion
* `--dev-url "sqlite://demo?mode=memory"` - Atlas relies on having an empty database called the
```

**Review by a8m — APPROVED** [review]

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+```
+atlas schema inspect --dev-url 'sqlite://demo?mode=memory&_fk=1' -u ent://ent/schema --visualize
+```
+
+![](https://entgo.io/images/assets/erd/edges-quick-summary.png)
+
+
+Hi Everyone!
+
+A few months ago we shared [entviz](/blog/2023/01/26/visualizing-with-entviz), a cool
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:23]:
```suggestion
A few months ago, we shared [entviz](/blog/2023/01/26/visualizing-with-entviz), a cool
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+tool that enables you to visualize your Ent schemas. Due to its success and popularity
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:24]:
```suggestion
tool that enables you to visualize your Ent schemas. Due to its success and popularity,
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+we decided to integrate it directly in [Atlas](https://atlasgo.io), the migration engine
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:25]:
```suggestion
we decided to integrate it directly into [Atlas](https://atlasgo.io), the migration engine
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+that Ent uses. 
+
+Since the release of [v0.13.0](https://atlasgo.io/blog/2023/08/06/atlas-v-0-13) of Atlas,
+you can now visualize your Ent schemas directly from Atlas without needing to install an
+additional tool.
+
+### Private vs. Public Visualizations
+
+Previously you could only share a visualization of your schema to the
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:34]:
```suggestion
Previously, you could only share a visualization of your schema to the
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+[Atlas Public Playground](https://gh.atlasgo.cloud/explore). While this is convenient
+for sharing your schema with others, it is not acceptable for many teams who maintain
+schemas that themselves are senstive and cannot be shared publicly.
+
+With this new release, you can easily publish your schema direclty to your private
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:39]:
```suggestion
With this new release, you can easily publish your schema directly to your private
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+For other installation options, see the [Atlas installation docs](https://atlasgo.io/getting-started#installation).
+
+Next, run the following command to generate a visualization of your Ent schema:
+
+```
+atlas schema inspect -u ent://ent/schema --dev-url 'sqlite://demo?mode=memory&_fk=1'  --visualize
+```
+
+Let's break this command down:
+* `atlas schema inspect` - this is command can be used to inspect schemas from a variety of sources and output
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:59]:
```suggestion
* `atlas schema inspect` – this command can be used to inspect schemas from a variety of sources and outputs
```

Comment (gedalyah-ariga) [reply doc/website/blog/2023-08-13-visualize-with-atlas.md:59]: I wasn't sure about "output" vs. "outputs," but I figured I would learn something if you said it was already correct @rotemtam

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+  them in various formats. In this case, we are using it to inspect an Ent schema.
+* `-u ent://ent/schema` - this is the URL to the Ent schema we want to inspect. In this case, we are using the
+  `ent://` schema loader to point to a local Ent schema in the `./ent/schema` directory.
+* `--dev-url 'sqlite://demo?mode=memory&_fk=1'` - Atlas relies on having an empty database called the
+  [Dev Database](https://atlasgo.io/concepts/dev-database) to normalize schemas and make various calculations.
+  In this case we are using an in memory SQLite database, but if you are using a different driver, you can use
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:65]:
```suggestion
In this case, we are using an in memory SQLite database; but, if you are using a different driver, you can use
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+```text
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Where would you like to share your schema visualization?:
+  ▸ Publicly (gh.atlasgo.cloud)
+    Your personal workspace (requires 'atlas login')
+```
+
+If you want to share your schema publicly, you can select the first option. If you want to share it privately, you
+can select the second option and then run `atlas login` to login to your (free) Atlas account.
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:78]:
```suggestion
can select the second option and then run `atlas login` to log in to your (free) Atlas account.
```

File: doc/website/blog/2023-08-13-visualize-with-atlas.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+### Wrapping up
+
+In this post we showed how you can easily visualize your Ent schema with Atlas. We hope you find this feature useful
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-13-visualize-with-atlas.md:82]:
```suggestion
In this post, we showed how you can easily visualize your Ent schema with Atlas. We hope you find this feature useful
```

**Pull request merged**